string event
string conditions
string format
string id
string size
---
string result
